<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewItemDlg</class>
 <widget class="QDialog" name="NewItemDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Item</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>41</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>41</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="lineTitle"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushOk">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>43</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>43</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="lineId"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="checkOpen">
     <property name="text">
      <string>Open file</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>41</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>41</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Status&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QComboBox" name="comboStatus"/>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineId</tabstop>
  <tabstop>lineTitle</tabstop>
  <tabstop>comboStatus</tabstop>
  <tabstop>checkOpen</tabstop>
  <tabstop>pushOk</tabstop>
  <tabstop>pushCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
